atomix/cluster/src/test/java/io/atomix/raft/snapshot/TestSnapshotStore.java

```diff
@@ -71,7 +71,7 @@ public void removeSnapshotListener(final PersistedSnapshotListener listener) {
   @Override
   public long getCurrentSnapshotIndex() {
     if (currentPersistedSnapshot.get() == null) {
-      return -1;
+      return 0;
```

Comment:
Ok. why :laughing:

Reply:
It is not clearly defined in the interface, but the actual implementation returns 0 when there is no snapshot. So the test implementation should also returns 0, otherwise it breaks the system. Somewhere in the code it is expecting the snapshotIndex to be non-negative.

Reply:
Ah ok thanks